app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetVetVisits.java

```diff
@@ -0,0 +1,21 @@
+package org.pesmypetcare.mypetcare.activities.fragments.infopet;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import org.pesmypetcare.mypetcare.databinding.FragmentInfoPetVetVisitsBinding;
+
+public class InfoPetVetVisits extends Fragment {
+    private FragmentInfoPetVetVisitsBinding binding;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'binding' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5464843)